css/main.css

```diff
@@ -35,54 +47,110 @@ nav {
 	padding: 0;
 }
 
-nav a {
-	text-decoration: none;
-	margin: 0;
+.column-1, .column-2 {
+	background-color: #ffffff;
+	display: none;
+	float: left;
+	top: 3em;
+	height: 6em;
+	width: 20%;
+	text-align: center;
 	color: #000000;
+	text-transform: upper-case;
 }
 
-li {
-	display: inline-block;
-	width: 250px;
-	text-align: center;
-	height: 40px;
+.column-1 div, .column-2 div,
+.column-3 div, .column-4 div {
+	height: 3em;
 	box-sizing: border-box;
-	padding-top: 10px;
+	padding: 1em 0;
+}
+
+.column-3, .column-4 {
 	background-color: #ffffff;
+	display: none;
+	float: right;
+	top: 3em;
+	height: 6em;
+	width: 20%;
+}
+
+.top-menu div a, .column-1 div a,
+.column-2 div a, .column-3 div a, 
+.column-4 div a {
+	text-decoration: none;
+	color: #000000;
+	text-transform: uppercase;
 }
 
-li:nth-child(3) {
-	margin-left: 185px;
+.top-menu div.item-1:hover div  {
+	display: block;
+}
 
+.top-menu div a:hover {
+	color: #91b2d0;
+}
+
+.top-menu div a:active {
+	color: #d6d0cc;
+}
+
+
+.top-menu div {
+	width: 20%;
+	float: left;
+	text-align: center;
+	height: 3em;
+	box-sizing: border-box;
+	background-color: #ffffff;
+	padding-top: 1em;
 }
 
 .logo {
-	display: inline-block;
+	display: block;
 	position: absolute;
-	left: 505px;
+	left: 50%;
+	margin-left: -8.5%;
```

Comment:
margin-left should be pixels because of the image
but also you can try to do it with **transform**